docs/src/theme/DocSidebar/ContextualContent/styles.module.css

```
@@ -0,0 +1,47 @@
+  font-weight: var(--ifm-font-weight-semibold);
+  text-align: center;
+  cursor: pointer;
+}
+
+.backArrow {
+  position: absolute;
+  left: 0;
+  transition: all 150ms ease-in-out;
+}
+
+@media (min-width: 997px) {
```

Stylelint (17.14.0)

[error] Expected "context" media feature range notation (media-feature-range-notation)

(media-feature-range-notation)